src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAX_VALUE_OF_PARAMETER = 25;
+
+    public static final int DEFAULT_RADIUS = 10;
+
+    public static final String DEFAULT_COLOR = "White";
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();
+
+    public Figure getRamdomFigure() {
+        int index = random.nextInt(5);
+        switch (index) {
+            case (0):
+                int randomRadius = random.nextInt(MAX_VALUE_OF_PARAMETER);
+                String randomColor = colorSupplier.getRandomColor();
+                return new Circle(randomColor, randomRadius);

```

Comment:
Move to private method, do the same for others
```suggestion
                return prepareCircle();
```